Makefile

```diff
@@ -154,3 +154,43 @@ scale-olm:
 undeploy:
 	hack/undeploy.sh
 .PHONY: undeploy
+
+
+# to use these targets, ensure the following env vars are set:
+# either each IMAGE env var:
+# IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY
+# IMAGE_ELASTICSEARCH_OPERATOR
+# IMAGE_ELASTICSEARCH6
+# IMAGE_LOGGING_KIBANA
```

Comment:
are the naming conventions different due to the underlying scripts?

Reply:
i was trying to use the names that api.ci would inject into the test jobs(if they were so configured) what naming conventions are you referring ti?

Reply:
i guess this should be logging_kibana6